workflow/rules/star.smk

```diff
@@ -0,0 +1,35 @@
+__author__ = "Jonas Almlöf"
+__copyright__ = "Copyright 2021, Jonas Almlöf"
+__email__ = "jonas.almlof@scilifelab.uu.se"
+__license__ = "GPL-3"
+
+
+rule star:
+    input:
+        fq1=lambda wildcards: alignment_input_fq1(wildcards),
```

Comment:
Ah, I get it now. Due to the use of the wrapper you need them separate. Is it not possible to get the separate from `alignment_input` though?

Reply:
Could you do something like:
```
fq1=alignment_input[0]
fq2=alignment_input[1]
```